test/modules/standard/jama/TestMatrix.chpl

```diff
@@ -0,0 +1,403 @@
+use Jama;
+use Math;
+
+/** TestMatrix tests the functionality of the Jama Matrix class and associated decompositions.
+<P>
+Run the test from the command line using
+<BLOCKQUOTE><PRE><CODE>
+ java Jama.test.TestMatrix 
```

Comment:
This command also seems to refer to the java version.  I'm guessing that doesn't apply to the Chapel file?